### PR TITLE
Wiki wide Writing Style Updates

### LINKED ISSUE #2637: Writing Style throughout the Wiki
Spelling and grammar updates and general writing style.

### PR DESCRIPTION
**Pull request by salmad3**
resolves #2637

### CODE REVIEW

File: docs/learn/learn-accounts.md [hunk 1]
```diff
@@ -12,48 +12,57 @@ on-chain. For a more in-depth explanation of the cryptography behind them, pleas
 
 ## Address Format
 
-The address format used in Substrate-based chains is SS58. SS58 is a modification of Base-58-check
-from Bitcoin with some minor modifications. Notably, the format contains an _address type_ prefix
-that identifies an address as belonging to a specific network.
+The address format used in Substrate-based chains is `SS58`. `SS58` is a modification of
+`Base-58-check` from Bitcoin with some minor changes. Notably, the form contains an _address type_
+prefix that identifies an address belonging to a specific network.
```
Comment (joepetrowski) [inline docs/learn/learn-accounts.md]: Make sure you merge the changes from 2663 into this - these changes are incorrect.

File: docs/learn/learn-accounts.md [hunk 2]
```diff
@@ -500,8 +515,8 @@ and sign the transaction.
 ![11](../assets/accounts/multisig-11.png)
 
 You will now see a pending transaction the 'Multisig Test Address' (the red '1' icon), and if you
-click the dropdown under Alice's balance, you will see that a value equivalent to the multisig
-deposit has been 'reserved', rendering that value untransferable until the multisig transaction
+click the dropdown under Alice's balance - you will see that a value equivalent to the multisig
+deposit has been 'reserved'; rendering that value untransferable until the multisig transaction
```
Comment (joepetrowski) [inline docs/learn/learn-accounts.md]: Why? Commas are correct here. Hyphen OK but semicolon is just wrong.

**Review by a-jwc — APPROVED** [review]

File: docs/learn/learn-runtime-upgrades.md [hunk 3]
```diff
@@ -63,11 +63,11 @@ changes to networking. It is essential to check the release notes, starting with
 priority and acting accordingly.
 
 General infrastructure providers, aside from following the Polkadot releases and upgrading in a
-timely manner, should monitor changes to runtime events and auxilliary tooling, such as the
+timely manner, should monitor changes to runtime events and auxiliary tooling, such as the
 [Substrate API Sidecar](https://github.com/paritytech/substrate-api-sidecar).
 
 Transactions constructed for runtime `n` will not work for runtimes `>n`. If a runtime upgrade
-occurs before broadcasting a priorly constructed transaction, you will need to reconstruct it with
+occurs before broadcasting a priory constructed transaction, you will need to reconstruct it with
```
Comment (joepetrowski) [inline docs/learn/learn-runtime-upgrades.md]: "priorly" is correct here. Or "previously"

File: docs/learn/learn-staking.md [hunk 4]
```diff
@@ -17,7 +17,7 @@ However, there is a probabilistic component to reward calculation (discussed bel
 not be exactly equal for all validators in a given era.
 
 Distribution of the rewards are pro-rata to all stakers after the validator payment is deducted. In
-this way, the network incents the nomination of lower-staked validators to create an equally-staked
+this way, the network incentivizes the nomination of lower-staked validators to create an equally-staked
```
Comment (joepetrowski) [inline docs/learn/learn-staking.md]: Better: "creates incentives".

File: docs/learn/learn-staking.md [hunk 5]
```diff
@@ -258,7 +258,7 @@ The UI shows it per validator and then all the affected nominators along with th
 unapplied, a governance proposal can be made to reverse it during this period (7 days on Kusama, 28
 days on Polkadot). After the grace period, the slashes are applied.
 
-The following levels of offence are
+The following levels of offense are
```
Comment (joepetrowski) [inline docs/learn/learn-staking.md]: "Offence" is more widely used internationally and is what is in the code, i.e. `pallet-offences`.

**Pull request merged**